apps/website/src/pages/docs/_layout.astro

```diff
@@ -106,6 +106,11 @@ const currentPage = new URL(Astro.request.url).pathname;
       grid-area: h1;
     }
 
+    :global(h2:not(left-sidebar)) {
```

Comment:
i don't think you need the `:not(left-sidebar)` part because this is already nested inside `.content-wrapper`

```suggestion
    :global(h2) {
```